src/integrations/morg-Slack-Bridge.md

```diff
@@ -56,9 +56,11 @@ An EMS server is not required.
 1. Slack is now added to the Matrix room  
 ![temp](/images/Screen%20Shot%202020-10-27%20at%204.51.41%20PM.png)
 
-1. Go to the channel you selected on Slack, and invite the riot bot with the command `/invite @riot_bridge`  
-![temp](/images/Screen%20Shot%202020-10-27%20at%204.56.16%20PM.png)
+1. Go to the channel you selected on Slack, and add the Element App via the `Integrations` tab in the members sidebar  
+![temp](/images/Screen%20Shot%202022-01-25%20at%203%2006%2048%20PM.png)  
```

Comment:
These three images doesn't load when running locally

![Screen Shot 2022-01-26 at 10 46 49 AM](https://user-images.githubusercontent.com/19155609/151149744-39f55a26-c48c-4f62-855d-b9d54d8155eb.png)

You only need to URI encode spaces as `%20`

```suggestion
![temp](/images/Screen%20Shot%202022-01-25%20at%203.06.48%20PM.png)  
```

(alternately, feel free to use more helpful file names and/or move the images to a dedicated directory per #8. If I were to start the wiki over today I would not use the default screenshot file names that require any URI encoding)